app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -8,12 +8,15 @@
 
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-      <%= render "govuk_publishing_components/components/title", {
-        context: title[:context],
-        title: title[:text],
-        inverse: true,
-        margin_top: 4
-      } %>
+        <%= render "govuk_publishing_components/components/title", {
+          context: title[:context],
+          title: title[:text],
+          inverse: true,
+          margin_top: 4
+        } %>
+      </div>
+      <div class="govuk-grid-column-one-third covid__logo-wrapper">
+        <%= render 'coronavirus_landing_page/components/shared/covid_logos' %>
```

Comment:
The image seems to be escaping from its bounding box, not sure why...

<img width="441" alt="Screenshot 2020-04-24 at 14 48 16" src="https://user-images.githubusercontent.com/861310/80219691-bddfb000-863a-11ea-9498-5faa843f5e2f.png">

Reply:
hmm - could be because of the `position` css